IVAN-MARAKHOVSKII/style.css

```diff
@@ -0,0 +1,122 @@
+body {
+  background-color: #515c65;
+  background-image: linear-gradient(to right, #33383c, #77828a, #33383c);
+}
+.grid {
+  height: 95vh;
+
+  display: grid;
+  grid-template: 24.2% 1fr 24.1% / 40.15% 1fr 39.6%;
+}
+.wheatherWidget {
+  grid-column-start: 2;
+  grid-row-start: 2;
+  
+  display: flex;
+  flex-direction: column;
+
+  background-color: #f1f8f9;
+  border-radius: 10px;
+}
+.refreshIcon {
+  width: 100%;
+  height: 20px;
+
+  margin-top: 16px;
+  margin-left: 351px;
+}
+.wheatherImage {
+  display: flex;
+  align-items: center;
+
+  margin: 32px 0px 53px 126px;
+}
+.infoBlock {
+  display: flex;
+}
+.tempIcon {
+  padding: 12px 0px 25px 23px;
+}
+.infoText {
+  padding: 11px 0px 14px 14px;
+
+  font-family: "Times New Roman", Times, serif;
+  letter-spacing: 1px;
+  color: #a2a9ad;
+}
+.infoTextTop {
+  font-size: 22px;
+  font-weight: 200;
+}
+.infoTextBottom {
+  font-weight: 900;
+  font-size: 14px;
+}
+.dayInfo {
+  height: 67px;
+  
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  
+  font-family: "Open Sans", sans-serif;
+  font-weight: 600;
+  letter-spacing: 1px;
+  color: white;
+  
+  margin-left: auto;
```

Comment:
Try to keep in mind order of properties, group them by type
1 - layout (position, display)
2 - anything related to box model (width/height/margin/padding)
3 - styling for element (color, background, box-shadow)
4 - anything related to font
5 - other stuff, it also includes overflow, z-index